File: android/src/me/isak/chess/AndroidLauncher.java
package me.isak.chess;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import me.isak.chess.Chess;

public class AndroidLauncher extends AndroidApplication {
	@Override
	protected void onCreate (Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
		Chess chess = new Chess();
		chess.play();	// To test that it is working
		initialize(chess, config);
	}
}

File: android/src/me/isak/chess/AndroidLauncher.kt
package me.isak.chess

import android.os.Bundle
import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration

class AndroidLauncher : AndroidApplication() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		val config = AndroidApplicationConfiguration()
		val androidInterface = AndroidInterface()
		val chess = Chess(androidInterface)
		chess.play()
		initialize(Chess(androidInterface), config)
	}
}
